tests/test_runtime_errors.py

```diff
@@ -92,11 +93,13 @@ def test_unable_to_reconstruct_message_pythonpath(monkeypatch: "MonkeyPatch") ->
         Pytest fixture, we use it to override ``PYTHONPATH``.
     """
     path = os.path.abspath(os.path.join("tests", "data", "notebook_for_testing.ipynb"))
-    message = f"Error reconstructing {path}"
+    message = re.escape(f"Error reconstructing {path}")
     monkeypatch.setenv("PYTHONPATH", os.path.join(os.getcwd(), "tests"))
-    with pytest.raises(RuntimeError) as excinfo:
+    monkeypatch.setattr(
+        nbqa.__main__, "_check_command_is_installed", lambda command: None
+    )
+    with pytest.raises(RuntimeError, match=message):
```

Comment:
I prefer `subprocess.run` solution.  That's what would happen at the end user side right?

Reply:
Yeah `subprocess.run` basically simulates the end-user having a `PYTHONPATH` set when running `nbqa remove_comments tests/data/notebook_for_testing.ipynb --nbqa-mutate`.